spyder_kernels/comms/commbase.py

```diff
@@ -393,18 +393,20 @@ def on_incoming_call(self, call_dict):
         if "pickle_highest_protocol" in call_dict:
             self._set_pickle_protocol(call_dict["pickle_highest_protocol"])
 
-    def _get_call_return_value(self, call_dict, call_data, comm_id):
-        """
-        Send a remote call and return the reply.
-
-        If settings['blocking'] == True, this will wait for a reply and return
-        the replied value.
-        """
+    def _send_call(self, call_dict, call_data, comm_id):
+        """Send call."""
         call_dict = self.on_outgoing_call(call_dict)
         self._send_message(
             'remote_call', content=call_dict, data=call_data,
             comm_id=comm_id)
 
+    def _get_call_return_value(self, call_dict, comm_id):
+        """_get_call_return_value
```

Comment:
```suggestion
        """
```

No need to have the function name here.